hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/fees/ConsensusServiceSimpleFeesSuite.java

```diff
@@ -0,0 +1,228 @@
+            return compare(() -> Arrays.asList(
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-txn"),
+                    validateChargedUsd("create-topic-txn", 0.01)));
+        }
+
+        @HapiTest()
+        @DisplayName("compare create topic with admin key")
+        final Stream<DynamicTest> createTopicWithAdminComparison() {
+            return compare(() -> Arrays.asList(
+                    getFileContents(SIMPLE_FEE_SCHEDULE).payingWith(GENESIS),
+                    newKeyNamed(ADMIN),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .adminKeyName(ADMIN)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-admin-txn"),
+                    validateChargedUsd("create-topic-admin-txn", 0.01630)));
+        }
+
+        @HapiTest()
+        @DisplayName("compare create topic with payer as admin key")
+        final Stream<DynamicTest> createTopicWithPayerAdminComparison() {
+            return compare(() -> Arrays.asList(
+                    newKeyNamed(ADMIN),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .adminKeyName(PAYER)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-admin-txn"),
+                    validateChargedUsd("create-topic-admin-txn", 0.01022)));
+        }
+
+        @LeakyHapiTest(overrides = {"fees.simpleFeesEnabled"})
+        @DisplayName("compare update topic with admin key")
+        final Stream<DynamicTest> updateTopicComparison() {
+            final String ADMIN = "admin";
+            return compare(() -> Arrays.asList(
+                    newKeyNamed(ADMIN),
+                    cryptoCreate(PAYER).balance(ONE_HUNDRED_HBARS),
+                    createTopic("testTopic")
+                            .blankMemo()
+                            .payingWith(PAYER)
+                            .adminKeyName(ADMIN)
+                            .fee(ONE_HBAR)
+                            .via("create-topic-admin-txn"),
+                    validateChargedUsd("create-topic-admin-txn", 0.01630),
+                    updateTopic("testTopic")
+                            .adminKey(ADMIN)
+                            .payingWith(PAYER)
+                            .fee(ONE_HBAR)
+                            .via("update-topic-txn"),
+                    validateChargedUsd("update-topic-txn", 0.000354)));
```

Comment:
Same here .. This should exactly be 0.00022 since it is a base update transaction